cfpb/cfgov-refresh
# Customize social media share link content

As part of the OAH updates, we want to individually customize the content for each of the social media share types on OAH Wagtail pages. This PR adds the ability to specify separate content in the Wagtail social media module for a page's email, LinkedIn, and Twitter links.
       
## Additions

- New fields for email, LinkedIn, and Twitter content

## Changes

- Use new content values in generating links in `social-media.html`

## Review
@schaferjh 
@higs4281 
@cfpb/cfgov-frontends  
@cfpb/cfgov-backends 

## Testing

- Create a page and add a social media block in its sidebar. 
- Add custom content for email, LinkedIn, and Twitter links.
- Preview page and verify that links work as expected.

## Screenshots

<img width="811" alt="screen shot 2016-11-30 at 8 58 40 am" src="https://cloud.githubusercontent.com/assets/778171/20762467/503cf90e-b6db-11e6-9381-acee49212e64.png">


<img width="678" alt="screen shot 2016-11-29 at 9 21 57 pm" src="https://cloud.githubusercontent.com/assets/778171/20761051/9531f5be-b6d6-11e6-9202-96d02327abd2.png">

<img width="555" alt="screen shot 2016-11-29 at 9 21 17 pm" src="https://cloud.githubusercontent.com/assets/778171/20761060/9afde5de-b6d6-11e6-9300-ea0f2dba65c0.png">

<img width="641" alt="screen shot 2016-11-29 at 9 20 35 pm" src="https://cloud.githubusercontent.com/assets/778171/20761066/9f60b25a-b6d6-11e6-926e-d9069aba6af4.png">


## Checklist
